kytos/cli/commands/napps/api.py

```diff
@@ -200,12 +200,14 @@ def list(cls, args):
         napps = [napp + ('[ie]',) for napp in mgr.get_enabled()]
         napps += [napp + ('[i-]',) for napp in mgr.get_disabled()]
 
-        # Sort, add description and reorder coloumns
+        # Sort, add description and reorder columns
         napps.sort()
         napps_ordered = []
         for user, name, status in napps:
-            napps_ordered.append((status, '{}/{}'.format(user, name),
-                                  mgr.get_description(user, name)))
+            description = mgr.get_description(user, name)
+            version = mgr.get_version(user, name)
```

Comment:
Done. KeyError is raised by [get_version()](https://github.com/kytos/kytos-utils/pull/109/files#diff-14c9ceec6fac0803fdc3fba43cf860d7R124).